src/gmp.c

```diff
@@ -26478,7 +26589,48 @@ gmp_xml_handle_text (/* unused */ GMarkupParseContext* context,
       APPEND (CLIENT_CREATE_REPORT_RR_RESULTS_RESULT_THREAT,
               &create_report_data->result_threat);
 
-
+	  case CLIENT_CREATE_REPORT_RR_RESULTS_RESULT_DETECTION_RESULT_DETAILS_DETAIL_NAME:
```

Comment:
This and the following added lines should use spaces instead of tabs.
Also, it would be nice if you could use the format
```
if (...)
  {
    ...
  }
```
instead of
```
if (...){
  ...
}
```